dataland-keycloak/realms/datalandsecurity-realm.json

```diff
@@ -1,6 +1,8 @@
 {
```

Comment:
Gibt es zu den Änderungen in dem realm.json hier irgendwas besonderes was man wissen sollte?

Sind das überhaupt aktive Änderungen von dir, oder hast du einfach einen neuen realm.export gemacht und die Datei committet, und das hier sind halt einfach die diffs?

Reply:
Ich habe nicht alle Änderungen übernommen (z.B. nicht die sich immer ändernden IDs) aber sonnst alles. Ich denke, dass die Änderungen einfach vom Version Update kommen